app/src/main/java/org/wikipedia/savedpages/SavedPageSyncService.java

```diff
@@ -65,7 +65,7 @@
     private SavedPageSyncNotification savedPageSyncNotification;
 
     public SavedPageSyncService() {
-        savedPageSyncNotification = SavedPageSyncNotification.getInstance();
```

Comment:
@sharvaniharan Notice the change you made here. Instead of getting the existing instance of this singleton class, you are now creating a new instance of it. This has introduced a bug. Please be mindful of these kinds of details.